docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  web:
+    container_name: web
+    build:
+      context: ./boilerplate
+      dockerfile: Dockerfile
+    ports:
+      - '4200:4200'
+      - '49153:49153'
+    networks:
+      - boilerplate-network
+    volumes:
+      - ./boilerplate:/usr/app
```

Comment:
This line will share the project with the container, to catch modifiers and trigger the live reload